server/controllers/metadata.controller.js

```diff
@@ -1,6 +1,7 @@
 import Metadata from '../models/metadata.model'
 import Marker from '../models/marker.model'
 import userCtrl from './user.controller'
+import APIError from '../helpers/APIError'
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import of 'APIError'. ([javascript:S1128](https://sonarcloud.io/organizations/aumanjoa-github/rules#rule_key=javascript%3AS1128))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=chronas-api&pullRequest=29&issues=AWbY54tdSgN4uCclP4pg&open=AWbY54tdSgN4uCclP4pg)

[ISSUE_KEY]: AWbY54tdSgN4uCclP4pg